examples/cross_val/scripts/cross_val_algo_random_forest.py

```diff
@@ -60,7 +92,7 @@
     # testtuple
     testtuple = client.add_testtuple({
         'traintuple_key': traintuple_key,
-        'data_manager_key': dataset_key,
+        'data_manager_key': data_manager_key,
```

Comment:
Due to the recent changes in the backend (optional evaluation), the objective key should be moved from the definition of the traintuple to the testtuple's.